src/deployment/efcore/tasks/README.md

```diff
@@ -27,6 +27,12 @@ Runs the EF Core migrations bundle.
       connectionString: $(ConnectionStrings.DatabaseContext)
 ```
 
+Note that this task must be run on a windows machine, which can be specified at the job, stage, or pipeline level:
```

Comment:
The task very much supports running on linux too, the OS to run the migration is dependent on the runtime used to build the bundle.

Perhaps say something like:

> This task can be run on both Windows and Linux agents. The OS you use will be based on the runtime that is used to build the migration bundle - i.e. if you build for the Linux runtime, you must run this task on a Linux agent.
>
> If you are using the `ef-migration-bundle.steps.yaml` template to build the bundle, the runtime can be configured with the `runtime` parameter, passing in either `windows` or `linux`.